slf4j/src/main/scala/org/typelevel/log4cats/slf4j/Slf4jFactory.scala

```diff
@@ -26,8 +27,23 @@ trait Slf4jFactory[F[_]] extends LoggerFactory[F] {
 }
 
 object Slf4jFactory extends LoggerFactoryGenCompanion {
+  @deprecated("LoggerFactories should be passed explicitly, not implicitly", "2.5.0")
   def apply[F[_]: Slf4jFactory]: Slf4jFactory[F] = implicitly
 
+  def create[F[_]: Sync]: Slf4jFactory[F] = new Slf4jFactory[F] {
```

Comment:
This would have been more convenient as an `apply` method, but unfortunately that's been squatted :(

Reply:
I think it's good this way! `cats.effect.std.Console` is also instatiated using `make` instead of `apply`. Maybe we should rename this to `make` rather than `create` :D

Reply:
Yes, agree now that we decided we are not totally deprecating `implicit` (otherwise, the `apply` was dead deprecated weight).

Regarding naming: I chose `create` based on this.
https://github.com/typelevel/log4cats/blob/4f02ef39fbee3568c9eaa52fee42904247988c62/slf4j/src/main/scala/org/typelevel/log4cats/slf4j/Slf4jLogger.scala#L38

But I think `make` is good too 🤔  and maybe better to keep it independent of `create`.